ref/general/basic-registry.adoc

```diff
@@ -63,6 +63,9 @@ You can then copy this encoded password into your `basicRegistry` element.
 
 This command can be used regardless of whether the registry is configured with the `basicRegistry` element or the `quickStartSecurity` element. For more information, see (link pending).
 
+== Federation of basic registries
+
+You can combine multiple basic user registries with each other, or with custom or LDAP registries, into a single federated registry. Federated registries provide a single logical view of multiple user registries and repositories. If multiple basic user registries are configured in your `server.xml` file or a `userRegistry.xml` file, the y ae confugred by default if you enable both the Application Security feature and the Federated User Registry feature.  For more information, see link:/docs/ref/feature/#federatedRegistry-1.0.html[the Federated User Registry feature].
```

Comment:
@dmuelle Just a typo in "the y ae confugred" I think is supposed to be "they are configured"

I also think that there's an extra space after the sentence that ends in "if you enable both the Application Security feature and the Federated User Registry feature."